audit/audit-gcp.sh

```diff
@@ -6,96 +6,151 @@ set -o pipefail
 
 CNCF_GCP_ORG=758905017065
 
-# gcloud organizations describe ${CNCF_GCP_ORG} 2>&1
-# ERROR: (gcloud.organizations.describe)
-# User [hh@ii.coop] does not have permission to access organization []
-
-format=json
-
 echo "# Auditing CNCF CGP Org: ${CNCF_GCP_ORG} #"
 mkdir -p org_kubernetes.io/roles
-for ROLE_PATH in $(gcloud iam roles list --organization="${CNCF_GCP_ORG}" --format="value(name)")
-do
+gcloud \
+    iam roles list \
+    --organization="${CNCF_GCP_ORG}" \
+    --format="value(name)" \
+| while read ROLE_PATH; do
     ROLE=$(basename "${ROLE_PATH}")
-    gcloud iam roles describe --organization="${CNCF_GCP_ORG}" "${ROLE}" \
-        --format=json > "org_kubernetes.io/roles/${ROLE}.json"
+    gcloud iam roles describe "${ROLE}" \
+        --organization="${CNCF_GCP_ORG}" \
+        --format=json \
+        > "org_kubernetes.io/roles/${ROLE}.json"
 done
-gcloud organizations get-iam-policy "${CNCF_GCP_ORG}" --format="${format}" \
-       > "org_kubernetes.io/iam.${format}"
+gcloud \
+    organizations get-iam-policy "${CNCF_GCP_ORG}" \
+    --format=json \
+    > "org_kubernetes.io/iam.json"
 
 echo "## Iterating over Projects ##"
-gcloud projects list \
-       --filter "parent.id=${CNCF_GCP_ORG}" \
-       --format "value(name, projectNumber)" \
-    | sort \
-    | while read PROJECT NUM; do
+gcloud \
+    projects list \
+    --filter="parent.id=${CNCF_GCP_ORG}" \
+    --format="value(name, projectNumber)" \
+| sort \
```

Comment:
FYI, sort is locale sensitive, so can behaves differently in cloudbuilder.  We likely want `export LC_ALL=C` if the order matters (not that it looks like you actually need a sort here)?

Reply:
I will remove 1 sort, the other is just for human-output as it iterates